.ado/windows-vs-pr.yml

```diff
@@ -51,6 +51,10 @@ jobs:
           BuildConfiguration: Release
           BuildPlatform: arm
           UseRNFork: true
+        Arm64Release:
+          BuildConfiguration: Release
+          BuildPlatform: arm64
+          UseRNFork: true
```

Comment:
Consider disabling the ArmRelease flavor, I don't know of any customer for this and we'll save precious disk space on the CI machines.